src/ipx.ts

```diff
@@ -123,10 +123,13 @@ class IPX {
   async getInfo ({ adapter, format, operations, src }: IPXImage): Promise<IPXImageInfo> {
     // Validate format
     if (format === '_') {
-      format = extname(src).substr(1)
+      const extension = extname(src).substr(1)
+      if (extension.match(/jpeg|webp|png|jpg|svg|gif/)) {
+        format = extension
```

Comment:
We may simply fallback fallback to `unknown` in L133 if guessed extension was wrong

Reply:
I've changed L133.  
This might be wrong because users may request invalid formats and still get results. 
```
/local/wired_format/_/image.jpg
```

Reply:
@pi0 Is it correct behavior?

Reply:
@farnabaz If cache key is fixed (always using unknown in key), i think yes